pkg/operator/sync.go

```diff
@@ -112,10 +115,20 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 	}
 	// as described by the name this is essentially static, but it no worse than what was here before.  Since changes disrupt workloads
 	// and since must perfectly match what the installer creates, this is effectively frozen in time.
-	kubeAPIServerServingCABytes, err := optr.getCAsFromConfigMap("openshift-config", "initial-kube-apiserver-server-ca", "ca-bundle.crt")
+	initialKubeAPIServerServingCABytes, err := optr.getCAsFromConfigMap("openshift-config", "initial-kube-apiserver-server-ca", "ca-bundle.crt")
 	if err != nil {
 		return err
 	}
+
+	// Fetch the following configmap and merge into the the initial CA. The CA is the same for the first year, and will rotate
+	// automatically afterwards.
```

Comment:
simple question: we are just auto-rotating whenever the cert expires?

Reply:
Yes... The openshift-kube-apiserver-operator will rotate the cert and update the configmap at roughly the 80% expiration of the cert (there is some jitter). This certificate expires after 1 year, so the update will happen somewhere around 9.6ish months.

Reply:
thanks for the explanation!